ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json

```diff
@@ -115,21 +115,29 @@
     },
     {
       "id" : "scientific_name",
-      "optional" : true,
-      "advanced" : true,
-      "allow_multiple" : false,
-      "default_values" : [ "" ],
-      "field_type" : "text",
-      "text_options" : {}
-    },
-    {
-      "id" : "taxon_id",
-      "optional" : true,
-      "advanced" : true,
+      "optional" : false,
```

Comment:
is `scientific_name` required now? why the uploader should enforce it while GFU allows it to be optional?

Reply:
Yes, because here we use scientific_name ui input to map the ncbi_taxon_id in the dynamic dropdown box.

Reply:
but why is it required now? Does GFU require `scientific_name`? If not, it should be optional.

Reply:
Do we still want to keep the `scientific_name` required for users? what if a user just doesn't know the scientific_name.

Reply:
Sorry...I should have set that to be optional=true.  It is done now.